readme.md

```diff
@@ -57,11 +80,72 @@ Filename. Mutually exclusive with the `extension` option.
 
 Get a temporary directory path. The directory is created for you.
 
+### tempy.directoryAsync()
+
+Returns a `Promise` with a temporary directory path. The directory is created for you.
+
+### tempy.clean()
+
+Get a list of deleted temporary directories and clears them. This is useful for when auto cleanup is disabled or when there are lots of temp files.
+
+### tempy.cleanAsync()
+
+Returns a `Promise` with a list of deleted temporary directories and clears them. This is useful for when auto cleanup is disabled or when there are lots of temp files.
+
+### tempy.jobFile(task, [options])
+
+Returns the value obtained in `task`.
+
```

Comment:
This should explain the benefit of the `job` type method (That it clean up after itself).